### PR TITLE
chore: add readme

### PR DESCRIPTION
**Pull request by 0xged**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+[![Lint](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/lint.yml/badge.svg?branch=main)](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/lint.yml)
+[![Tests](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/tests.yml/badge.svg?branch=main)](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/tests.yml)
+[![Slither Analysis](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/slither.yml/badge.svg?branch=main)](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/slither.yml)
+
+# StaticOracle
+
+StaticOracle is a tool developed under Uniswap's grant program that aims to help developers integrate easily and fast with uniswap's v3 twaps oracles.
+
+This is done by leveraging already existing [WeightedOracleLibrary]() implemented by [Nicolás Chamo](https://github.com/nchamo).
```
Comment (0xsambugs) [inline README.md:9]: Where is `WeightedOracleLibrary` pointing to?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+[![Lint](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/lint.yml/badge.svg?branch=main)](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/lint.yml)
+[![Tests](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/tests.yml/badge.svg?branch=main)](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/tests.yml)
+[![Slither Analysis](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/slither.yml/badge.svg?branch=main)](https://github.com/Mean-Finance/uniswap-v3-oracle/actions/workflows/slither.yml)
+
+# StaticOracle
+
+StaticOracle is a tool developed under Uniswap's grant program that aims to help developers integrate easily and fast with uniswap's v3 twaps oracles.
```
Comment (0xsambugs) [inline README.md:7]:
```suggestion
StaticOracle is a tool developed under Uniswap's grant program that aims to help developers integrate easily and fast with Uniswap's v3 TWAP oracles.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+This is done by leveraging already existing [WeightedOracleLibrary]() implemented by [Nicolás Chamo](https://github.com/nchamo).
+
+StaticOracle will allow developers to:
+
+- Prepare a set of pools (for exmaple: by a token pair and a fee tier) to support a certain period of time (for example: 60 seconds).
```
Comment (0xsambugs) [inline README.md:13]:
```suggestion
- Prepare a set of pools (for example: by a token pair and a fee tier) to support a certain period of time (for example: 60 seconds).
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+- Quote a TWAP for a set of pools (for exmaple: by a token pair and a fee tier, or addresses).
```
Comment (0xsambugs) [inline README.md:14]:
```suggestion
- Quote a TWAP for a set of pools (for example: by a token pair and a fee tier, or addresses).
```

**Review by 0xsambugs — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (0xged) [reply README.md:9]: Fixed, to https://github.com/Uniswap/v3-periphery/pull/146

**Pull request merged**